### DISCUSSION TITLE
Place any question here.

### DISCUSSION THREAD

**Discussion by pajoma** [pajoma/vscode-journal#69] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blonewolfs** [comment]
Is it possible to change the template for a scoped entry?  At my job I'm part of an on-call rotation and have a set of tasks that are done only when it is my turn. So, it would be nice to be able to do something like '#on-call' and have it populate with the task list rather than the default '## tasks ##notes' file.

Thank you.
